ocaml/idl/datamodel_cluster.ml

```diff
@@ -28,10 +38,25 @@ let create = call
     ~doc:"Creates a Cluster object and one Cluster_host object as its first member"
     ~result:(Ref _cluster, "the new Cluster")
     ~versioned_params:
-            ([{param_type=Ref _network; param_name="network"; param_doc="the single network on which corosync carries out its inter-host communications"; param_release=kolkata_release; param_default=None};
-             {param_type=String; param_name="cluster_stack"; param_doc="simply the string 'corosync'. No other cluster stacks are currently supported"; param_release=kolkata_release; param_default=None};
-             {param_type=Bool; param_name="pool_auto_join"; param_doc="true if xapi is automatically joining new pool members to the cluster"; param_release=kolkata_release; param_default=None};
-      ] @timeout_params)
+      ([{param_type=(Ref _pif);
+         param_name="PIF";
+         param_doc="The PIF to connect the cluster's first cluster_host to";
+         param_release=lima_release;
+         param_default=None};
+
+        {param_type=String;
+         param_name="cluster_stack";
+         param_doc="simply the string 'corosync'. No other cluster stacks are currently supported";
+         param_release=kolkata_release;
+         param_default=None};
+
+        {param_type=Bool;
+         param_name="pool_auto_join";
+         param_doc="true if xapi is automatically joining new pool members to the cluster";
```

Comment:
I know this was already there, but still: don't use terms like "xapi" here. This goes into the API docs that should be independent of the implementation, daemons etc.

Better is something like "Indicates whether new pool members are automatically added to the cluster".